### DISCUSSION TITLE
expired due to inactivity livewire in package

### DISCUSSION THREAD

**Discussion by infohojin** [livewire/livewire#3361] [1. Help] [answered]
I make sample counter code by livewire. and this component add routing /routes/web,php

 ex) /routing/web.php
```
use App\Http\Livewire\Counter;
Route::get('/counter', Counter::class);
```
==> it is ok.

move this code my package route config. it is not working
ex)/mypackage/module/routung,php

error message is "this page has expired due to inactivity"

I don't know, why this is diffrent?

**Comment by infohojin** [comment, marked as answer]
i solved.
laravel  customer package is not apply 'web' web middleware. and session is not started.

i changed web.php

```
Route::prefix('jinyui')->group(function () {
    Route::get('/', function () {
        return view("jinyui::demo.index");
    });
});
```

**Reply by StevieDC** [reply]
For anyone needing this. The above didn't work for me but the following did ...
`
    Route::get('/lwtest', function () {
        return view("wordpackage::lwtest");
    })->middleware('web');
`
